doc/api/net.md

```diff
@@ -3,20 +3,54 @@
 > Stability: 2 - Stable
 
 The `net` module provides an asynchronous network API for creating stream-based
-servers ([`net.createServer()`][]) and clients ([`net.createConnection()`][])
-that implement TCP or local communications (domain sockets on UNIX, named pipes
-on Windows). It can be accessed using:
+TCP or [IPC][] servers ([`net.createServer()`][]) and clients
+([`net.createConnection()`][]).
+
+It can be accessed using:
 
 ```js
 const net = require('net');
 ```
 
+## IPC Support
+
+The `net` module supports IPC with named pipes on Windows, and domain sockets
+on other operating systems.
+
+### Identifying paths for IPC connections
+
+[`net.connect()`][], [`net.createConnection()`][], [`server.listen()`][] and
+[`socket.connect()`][] takes a `path` parameter to identify IPC endpoints.
+
+On UNIX, the local domain is usually known as the UNIX domain. The path is a
+filesystem path name. It gets truncated to `sizeof(sockaddr_un.sun_path) - 1`,
+which varies on different operating system between 91 and 107 bytes.
+The typical values are 107 on Linux and 103 on OS X. The path is
+subject to the same naming conventions and permissions checks as would be done
+on file creation. It will be visible in the filesystem, and will *persist until
+unlinked*.
+
+*Note*: Unix named pipes (FIFOs) are not supported.
```

Comment:
Yes it does look stranger if pulled out to here, where the context of `net` being about bidirectional streams is clearer. This can be removed.